### DISCUSSION TITLE
What is the recommended way to list the namespaced Deployments?

### DISCUSSION THREAD

**Discussion by 64J0** [kubernetes-client/csharp#1521] [Q&A] [answered]
Hello, hope you're good. I noticed that there is a function to list namespaced Pods (`CoreV1.ListNamespacedPod`), but there is no function to list namespaced [Deployments](https://kubernetes.io/docs/concepts/workloads/controllers/deployment/).

What is the recommended way to list the namespaced Deployments then?

I'm using KubernetesClient package version `13.0.11`.

**Comment by 64J0** [comment, marked as answer]
Just found out my mistake. It's not inside the `CoreV1`.

One must use the `Kubernetes` object directly.

* Reference: https://stackoverflow.com/questions/66297311/kubernetes-client-c-sharp-kubectl-get-services

**Reply by tg123** [reply]
it is in appsv1
obj impled all interfaces that is why you see it
